Front/addsavenote.cpp

```diff
@@ -147,3 +148,19 @@ void AddSaveNote::on_SaveNoteButton_clicked()
 
 }
 
+
+void AddSaveNote::on_ASNCanselButton_clicked()
+{
+    Category* w = new Category();
+    w->show();
+    close();
+}
+
+
+void AddSaveNote::on_ASNHomeButton_clicked()
+{
+    Category* w = new Category();
```

Comment:
Same here, consider reading [this](https://www.geeksforgeeks.org/smart-pointers-cpp/) or watching [this](https://www.youtube.com/watch?v=xGDLkt-jBJ4)

Reply:
Fixed